### PR TITLE
Feature/syal 1253

### PR DESCRIPTION
**Pull request by HieuLM-AVISPL**

### CODE REVIEW

File: src/main/java/com/avispl/symphony/dal/infrastructure/management/tripleplay/TriplePlayAggregatorCommunicator.java [hunk 1]
```diff
@@ -0,0 +1,714 @@
...
+
+	@Override
+	protected void authenticate() throws Exception {
+
+	}
+
+	@Override
+	public List<AggregatedDevice> retrieveMultipleStatistics() throws Exception {
+		if (logger.isWarnEnabled()) {
+			logger.warn("Start call retrieveMultipleStatistic");
```
Comment (MaksimR-AVISPL) [inline src/main/java/com/avispl/symphony/dal/infrastructure/management/tripleplay/TriplePlayAggregatorCommunicator.java:276]: Is warning level necessary here? This is what would happen normally, so maybe warning is a bit too much :)

File: src/main/java/com/avispl/symphony/dal/infrastructure/management/tripleplay/TriplePlayAggregatorCommunicator.java [hunk 2]
```diff
@@ -0,0 +1,714 @@
...
+	/**
+	 * update service in cached clients
+	 *
+	 * @param client client to update
+	 * @param cachedClient client in cached clients list need to update
+	 * @param value name of service to set
+	 */
+	private void updateServiceInCachedClients(AggregatedDevice client, Client cachedClient, String value) {
+		for (AdvancedControllableProperty advancedControllableProperty : client.getControllableProperties()) {
+			if (advancedControllableProperty.getName() == buildNameForChannel(ChannelMetric.SELECT_CHANNEL.getName())) {
```
Comment (MaksimR-AVISPL) [inline src/main/java/com/avispl/symphony/dal/infrastructure/management/tripleplay/TriplePlayAggregatorCommunicator.java:672]: do you want to compare the references here, or should it be .equals() instead?

File: src/main/java/com/avispl/symphony/dal/infrastructure/management/tripleplay/TriplePlayAggregatorCommunicator.java [hunk 3]
```diff
@@ -0,0 +1,714 @@
...
+				client.getProperties().put(buildNameForChannel(ChannelMetric.LAST_CHANNEL.getName()), advancedControllableProperty.getValue().toString());
+				break;
+			}
+		}
+		for (Service service : cachedClient.getServices()) {
+			if (service.getName() == client.getProperties().get(buildNameForChannel(ChannelMetric.LAST_CHANNEL.getName()))) {
```
Comment (MaksimR-AVISPL) [inline src/main/java/com/avispl/symphony/dal/infrastructure/management/tripleplay/TriplePlayAggregatorCommunicator.java:678]: service.getName() is also a string, don't you need equals() here and below?

**Review by MaksimR-AVISPL — APPROVED** [review]

File: src/main/java/com/avispl/symphony/dal/infrastructure/management/tripleplay/TriplePlayAggregatorCommunicator.java (hunk 1 above)
Comment (HieuLM-AVISPL) [reply src/main/java/com/avispl/symphony/dal/infrastructure/management/tripleplay/TriplePlayAggregatorCommunicator.java:276]: I fixed it

File: src/main/java/com/avispl/symphony/dal/infrastructure/management/tripleplay/TriplePlayAggregatorCommunicator.java (hunk 3 above)
Comment (HieuLM-AVISPL) [reply src/main/java/com/avispl/symphony/dal/infrastructure/management/tripleplay/TriplePlayAggregatorCommunicator.java:678]: I fixed it

File: src/main/java/com/avispl/symphony/dal/infrastructure/management/tripleplay/TriplePlayAggregatorCommunicator.java (hunk 2 above)
Comment (HieuLM-AVISPL) [reply src/main/java/com/avispl/symphony/dal/infrastructure/management/tripleplay/TriplePlayAggregatorCommunicator.java:672]: I fixed it

**Pull request merged**